### ISSUE TITLE
Keeping individual orbit(s) visible when body is not selected

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#1122]
I would like to be able to show an individual body's orbit even after I have cleared the selection, much like the "isolate constellation" feature works. I guess there is currently no way of doing that, so I file this as a feature request.

**Comment by alex-w** [comment]
Please describe reasons for requesting a new feature, e.g. planned use cases.

**Comment by Atque** [comment]
@alex-w 
Let's say I want to view the orbit of Uranus, and then I happen to select Antares or some other star for some reason. Suddenly, all orbits show. It would be nice to keep Uranus' orbit, until you hit a certain key or another SSO (which would show that one's orbit too). It could also be nice to show for example only Earth's and Venus' orbits when watching from far away in space.

Compare this to the isolate constellation feature.

**Comment by alex-w** [comment]
@Atque please check 23.3RC3 - @gzotti updated the code for orbit rendering

**Comment by Atque** [comment]
Hmm, not quite what the issue asks for. I had in mind something similar to the isolate constellation feature.

**Comment by gzotti** [comment]
Yes, the current (refreshed) behaviour is what has been requested so far, no preemptive magic :-)
The logic in the current implementation is very strict. Once you lose selection, the other rules apply. 
Given how difficult the single-constellation thingy is to comprehend, I am also unsure whether this should be implemented. I'd rather add "colored main planet orbits plus colored-by-type others". 

We would need to have some sub-panels popping up with a dynamic list of all currently existing SSOs wich checkboxes from which you can select which orbits should be plotted. Same goes with constellations, although I see even more need in doing that...

**Comment by alex-w** [comment]
@Atque what about release 24.3?

**Comment by gzotti** [comment]
Not yet. We can now show selected object's orbit plus the planets. This works also on SSObserver, but when you don't focus Sun, movement is still awkward.  My plan is to have a selection option (GUI similar to RemoteSync property exclusion selection) where you can have a permanent selection option of 3-10 favourite objects (a few comets, a few asteroids, ...). But it takes some free weeks...
